### DISCUSSION TITLE
How to get trainset for CE?

### DISCUSSION THREAD

**Discussion by lhycms** [CederGroupHub/smol#472] [Q&A]
Hi, 
Could you know how to generate a initial trainset to fit CE? 
Maybe we can use MC+DFT, or just use SQS+DFT?

Any help is appreciated!

**Comment by lbluque** [comment]
Hi @lhycms,

There are many ways for generating an initial training set for CE. Some of the simplest involve exhaustive enumeration of symmetrically distinct configurations, random structure generation, or simulated annealing with a simple pair potential.  As you mention you can also generate SQS if you are planning to study high entropy or disordered systems. 

I recommend that you look into the literature since there are plenty of methods used in different studies.
